src/main/java/alexei_drujinin/Arrays.java

```diff
@@ -0,0 +1,61 @@
+package alexei_drujinin;
+
+import java.util.Scanner;
+
+public class Arrays {
+
+    public static void main(String[] args) {
```

Comment:
very long main method, separate it on sub methods to make it easier to read

Reply:
done